var/spack/repos/builtin/packages/google-cloud-cli/package.py

```diff
@@ -51,7 +85,11 @@ class GoogleCloudCli(Package):
 
     depends_on("c", type="build")  # generated
 
-    depends_on("python", type=("build", "run"))
+    with default_args(type=("build", "run")):
+        depends_on("python@3.8:3.13", when="@500:")
+        depends_on("python@3.8:3.12", when="@456:499")
+        depends_on("python@3.8:3.10", when="@488:")
+        depends_on("python@3.5:", when="@427:")
```

Comment:
Minimum (non-deprecated) version of python in spack is now 3.9, so these are all satisfied by default:
```suggestion
        depends_on("python")
        depends_on("python@:3.13", when="@500:")
        depends_on("python@:3.12", when="@456:499")
        depends_on("python@:3.10", when="@:455")
```
With these changes, merge conflict will be easier to resolve as well.